u-root.go

```diff
@@ -272,10 +277,8 @@ func Main(l ulog.Logger, buildOpts *gbbgolang.BuildOpts) error {
 	if !*noCommands {
 		var b builder.Builder
 		switch *build {
-		case "bb":
```

Comment:
why are we removing bb?

Reply:
I thought that was the plan, to only rely on gobusybox instead of the u-root internal code? Also I merged the bb and gbb as they would do the same thing, providing bb will still work but will have the same effect as gbb.